src/core/index.js

```diff
@@ -157,7 +157,9 @@ export function stopRendering(m) {
 function extractUIOptions(id, options) {
   const closable = options.container
     ? false
-    : undefined === options.closable ? true : !!options.closable;
+    : undefined === options.closable
+      ? true
+      : !!options.closable;
```

Comment:
I did not change this line. I don't know where the change came from.

Reply:
This is just our prettifier automatically breaking the line